LifeCycleAssessment_Engine/Compute/EvaluateEnvironmentalProductDeclaration.cs

```diff
@@ -93,7 +96,7 @@ public static LifeCycleAssessmentElementResult EvaluateEnvironmentalProductDecla
                         return null;
                     case QuantityType.Volume:
                         BH.Engine.Base.Compute.RecordNote("Evaluating object type: " + elementM.GetType() + " based on EPD Volume QuantityType.");
-                        var evalByVolume = EvaluateEnvironmentalProductDeclarationByVolume(elementM, phases, field, exactMatch);
+                        var evalByVolume = EvaluateEnvironmentalProductDeclarationByVolume(elementM, phases, mc, field, exactMatch);
                         value += evalByVolume.Quantity;
                         if (resultValue == null)
                             resultValue = evalByVolume;
```

Comment:
_comment actually refers to line 114._
This should be updated to call to the new GetElementEPD method with MaterialComposition included to eliminate the redundant MaterialComposition call within the currently called method.